app/controllers/products_controller.rb

```diff
@@ -2,22 +2,17 @@ class ProductsController < ApplicationController
   layout "application"
   before_action :set_product, only: [:show, :edit, :update, :destroy]
```

Comment:
shouldn't have a edit/update/destroy though because this is the customer facing controller